src/Magnum/OvrIntegration/HmdEnum.h

```diff
@@ -83,6 +86,183 @@ typedef Containers::EnumSet<HmdTrackingCapability> HmdTrackingCapabilities;
+     * to make sure the user is in a comfortable pose, facing a comfortable
+     * direction.
+     */
+    EyeLevel = ovrTrackingOrigin_EyeLevel,
+
+    /**
+     * @brief Tracking system origin reported at floor height
+     *
+     * Prefer using this origin when your application requires the physical
+     * floor height to match the virtual floor height, such as standing
+     * experiences.
+     * When used, all poses in ovrTrackingState are reported as an offset
+     * transform from the profile calibrated floor pose. Calling @ref Hmd::recenterTrackingOrigin()
+     * will recenter the X & Z axes as well as yaw, but the Y-axis (i.e. height)
+     * will continue to be reported using the floor height as the origin for
+     * all poses.
+     */
+    FloorLevel = ovrTrackingOrigin_FloorLevel,
+};
+
+/**
+@brief Tracker flag
+
+@see @ref TrackerFlags
+*/
+enum class TrackerFlag: Int {
+    /**< The sensor is present, else the sensor is absent or offline */
+    Connected = ovrTracker_Connected,
+
+    /**
+     * The sensor has a valid pose, else the pose is unavailable.
+     * This will only be set if ovrTracker_Connected is set.
+     */
+    PoseTracked = ovrTracker_PoseTracked
+};
+
+/**
+@brief Tracker flags
+*/
+typedef Containers::EnumSet<TrackerFlag> TrackerFlags;
+
+CORRADE_ENUMSET_OPERATORS(TrackerFlags)
+
+/**
+@brief Button
+
+The Oculus SDK provides an abstraction of input for the XBox controller,
+the Oculus Remote and Oculus Touch. This enum describes all possible buttons
+pressable on one of these controllers.
+
+@see @ref Buttons
+*/
+enum class Button: UnsignedInt {
+    A = ovrButton_A,
+    B = ovrButton_B,
+
+    X = ovrButton_X,
+    Y = ovrButton_Y,
+
+    RThumb = ovrButton_RThumb,
+    RShoulder = ovrButton_RShoulder,
+    LThumb = ovrButton_LThumb,
+    LShoulder = ovrButton_LShoulder,
+
+    Up = ovrButton_Up,          /**< D pad up */
+    Down = ovrButton_Down,      /**< D pad down */
+    Left = ovrButton_Left,      /**< D pad left */
+    Right = ovrButton_Right,    /**< D pad right */
+
+    Enter = ovrButton_Enter,    /**< Start on XBox controller */
+    Back = ovrButton_Back,      /**< Back on Xbox controller */
+
+    VolUp = ovrButton_VolUp,    /**< Only supported by Remote */
+    VolDown = ovrButton_VolDown,/**< Only supported by Remote */
+
+    Home = ovrButton_Home,
+};
+
+/**
+@brief Buttons
+*/
+typedef Containers::EnumSet<Button> Buttons;
+
+CORRADE_ENUMSET_OPERATORS(Buttons)
+
+/** @brief Bit mask of all buttons on the right Touch controller */
+constexpr Buttons R_BUTTON_MASK = Button::A | Button::B | Button::RThumb | Button::RShoulder;
+
+/** @brief Bit mask of all buttons on the left Touch controller */
+constexpr Buttons L_BUTTON_MASK = Button::X | Button::Y | Button::LThumb | Button::LShoulder;
+
+/** @brief Bit mask of buttons used by Oculus Home */
+constexpr Buttons PRIVATE_MASK = Button::VolUp | Button::VolDown | Button::Home;
```

Comment:
Oh crap.

Since the type isn't really needed (but still would be nice to have it there for Doxygen):
```cpp
struct Buttons: Containers::EnumSet<Button> {
    constexpr 
        #ifdef DOXYGEN_GENERATING_OUTPUT
        Buttons
        #else
        Containers::EnumSet<Button>
        #endif
        LMask = ...;

    using EnumSet::EnumSet; // inherit the constructors
};
```